site/app/templates/misc/MarkdownArea.twig

```diff
@@ -65,5 +66,6 @@
 <script>
     $('#{{markdown_area_id}}').ready(function() {
         enableTabsInTextArea("#{{markdown_area_id}}");
-    })
+    });
+    $('#{{markdown_area_id}}').closest('.markdown-area').find('[data-initialize_preview="1"]').trigger('click');
```

Comment:
based on [this explanation of `.ready()`](http://tutorials.jenkov.com/jquery/document-ready.html ), I think this line of code should be in there to make sure the button has loaded first?

Reply:
Technically, `.ready()` is only necessary for scripts that are in the `<head>` tag or earlier in the `<body>` tag than the elements they try to reference, since the DOM loads in a linear fashion so it is guaranteed that the top part of the twig template would render before the script tag is executed.

But I do agree, it definitely doesn't hurt to add, plus I already added it on the line above. Thanks for catching this!

Reply:
Ohh I see, thanks for sharing about the DOM loading! You can make the call for what's best, or leave it as is then.